src/sentry/static/sentry/less/shared-components.less

```diff
@@ -447,37 +447,25 @@ table.integrations {
     padding-left: 78px !important;
   }
 
-  .integration-icon {
-    position: absolute;
-    top: 15px;
-    left: 16px;
-    background: @blue-light;
-    .square(48px);
-    border-radius: 2px;
-    color: #fff;
-
-    &:after {
-      display: block;
-      font-family: "sentry-simple";
-      content: "\e622";
-      font-size: 24px;
+  .icon-integration {
+      position: absolute;
+      top: 15px;
+      left: 16px;
       .square(48px);
-      text-align: center;
-      line-height: 48px;
-    }
+      border-radius: 2px;
+      border: 0;
+      /* this is so that there aren't gray boxes before load */
+      background-color: #fff;
+      display: inline-block;
   }
 
   .auto-tag-browsers,
   .auto-tag-device,
   .auto-tag-interface-types,
   .auto-tag-operating-systems,
   .auto-tag-urls {
-    .integration-icon {
-      background: @orange;
-
-      &:after {
-        content: "\e603";
-      }
+    .icon-integration {
+      .icon-sentry;
```

Comment:
As far as I can tell, this is a LESS thing that allows the surrounding class to assume the style of the class declared inside the braces